ndpi-netfilter/ipt/libxt_ndpi.c

```diff
@@ -800,9 +800,10 @@ void _init(void)
 			   vs = strstr(buf,"#version");
 			   if(!vs)
 	    			xtables_error(PARAMETER_PROBLEM, "xt_ndpi: #version missing");
-			   if(!strstr(vs+8,NDPI_GIT_RELEASE))
-	    			xtables_error(PARAMETER_PROBLEM, "xt_ndpi: module version %s != %s",
-						vs+8,NDPI_GIT_RELEASE);
+			   // TODO - skipping ndpi version
```

Comment:
We can close PR 1 and solve this here.  I think we should.  (Adding all the protocols was so easy, we have lots of time left in the ticket to address this. ;-) )